src/Entity/Jecoute/Choice.php

```diff
@@ -29,7 +29,6 @@ class Choice
      * @var Question
      *
      * @ORM\ManyToOne(targetEntity="Question", inversedBy="choices")
-     * @ORM\JoinColumn(onDelete="CASCADE")
```

Comment:
so we won't be able to delete any Question that has a Choice.
do you confirm? (we don't delete Questions, right?)